src/imagecaption/imagecaptionengine.js

```diff
@@ -50,15 +52,13 @@ export default class ImageCaptionEngine extends Plugin {
 		 * @member {module:engine/view/editableelement~EditableElement} #_lastSelectedCaption
 		 */
 
-		this._viewCaptionsToUpdate = [];
```

Comment:
This is unused already and this is why it's removed. Somehow passed in previous PRs.